### ISSUE TITLE
wa-flank:end not wrapping on mobile with multi-element filter section

### ISSUE THREAD

**Issue by devhassan941-png** [shoelace-style/webawesome#1916]
I have a wa-flank:end layout containing a filter section (with multiple form inputs: week picker + employee dropdown) on the left and a button on the right. On desktop it works perfectly, but on mobile the layout doesn't wrap to vertical stack as expected - instead, the date text in the week picker wraps internally to multiple lines, making it look cramped and awkward.

```
<div class="wa-flank:end" style="--content-percentage: 20%;">
    <div>
        <!-- Filter section with multiple inputs -->
        <r-week-picker name="weekStart" value="2025-12-29"></r-week-picker>
        <wa-select name="employeeId">
            <wa-option>Employee Name</wa-option>
        </wa-select>
    </div>
    <wa-button variant="brand">New Row</wa-button>
</div>

```
On mobile (narrow screens), the wa-flank:end should automatically wrap to a vertical stack:
- Filter inputs on top
- Button below

**On mobile:**
- The week picker shrinks horizontally and expands vertically (date text wraps to multiple lines)
- wa-flank:end doesn't detect the content is being squeezed
- Layout stays horizontal with cramped, wrapped text
<img width="1079" height="1990" alt="Image" src="https://github.com/user-attachments/assets/a08ece00-9f35-40c9-b975-101266f912e1" />

**What I've Tried:**

1. Adjusted --content-percentage: Tried values from 50% down to 5% - no change in wrapping behavior
2. Added min-width to week picker: This caused elements to overlap/overflow but still didn't trigger wrapping
<img width="425" height="699" alt="Image" src="https://github.com/user-attachments/assets/fae4e800-1969-4b6c-ba9f-08a2eec5b961" />

What's the recommended way to make wa-flank:end wrap to vertical stack on mobile when the content contains multiple flexible form elements? Is there a CSS custom property or technique I'm missing, or do I need a different layout utility for this use case?

Thanks for any guidance!

**Comment by gregjotau** [comment]
@claviska not sure if this one should maybe be moved to discussions? We would really like guidance on this one :)

**Comment by kelseythejackson** [comment]
@gregjotau Thanks for raising this issue. I believe this is working as designed, but I can see how the behavior of `--content-percentage` might be counterintuitive.                                                                                                                                                                                                                

  ## How `wa-flank:end` wrapping works                                                                                                                                                                                                                                                                                                                                     
   
  The `--content-percentage` variable sets the **minimum width** of the content area (the first child). Wrapping is triggered when:                                                                                                                                                                                                                                        
                  
  content minimum width + flank width > container width

  ## The counterintuitive part

  Lowering `--content-percentage` actually makes wrapping *less* likely, not more. Here's why:

  | `--content-percentage` | 320px screen | Content min | Result |
  |------------------------|--------------|-------------|--------|
  | `20%` | 320px | 64px | Plenty of room, no wrap |
  | `70%` | 320px | 224px | Forces wrap much sooner |

  ## Solution

  To get earlier wrapping on mobile, **increase** the percentage:

  ```html
  <div class="wa-flank:end" style="--content-percentage: 70%;">
    <div>
      <r-week-picker name="weekStart" value="2025-12-29"></r-week-picker>
      <wa-select name="employeeId">
        <wa-option>Employee Name</wa-option>
      </wa-select>
    </div>
    <wa-button variant="brand">New Row</wa-button>
  </div>
  ```
  You may need to experiment with values between 60-80% to find the breakpoint that works best for your content.
